src/main/java/mate/academy/Main.java

```diff
@@ -44,12 +54,33 @@ public static void main(String[] args) {
         yesterdayMovieSession.setMovie(fastAndFurious);
         yesterdayMovieSession.setShowTime(LocalDateTime.now().minusDays(1L));
 
-        MovieSessionService movieSessionService = null;
+        MovieSessionService movieSessionService = (MovieSessionService)
+                injector.getInstance(MovieSessionService.class);
         movieSessionService.add(tomorrowMovieSession);
         movieSessionService.add(yesterdayMovieSession);
 
         System.out.println(movieSessionService.get(yesterdayMovieSession.getId()));
         System.out.println(movieSessionService.findAvailableSessions(
                         fastAndFurious.getId(), LocalDate.now()));
+
+        AuthenticationService authenticationService = (AuthenticationService)
+                injector.getInstance(AuthenticationService.class);
+
+        User exampleUser = null;
+        try {
+            exampleUser = authenticationService.register("user@gmail.com", "123455667");
+        } catch (RegistrationException e) {
+            System.out.println("Can`t register user: " + e);
+        }
+        ShoppingCartService shoppingCartService =
+                (ShoppingCartService) injector.getInstance(ShoppingCartService.class);
+        shoppingCartService.addSession(tomorrowMovieSession, exampleUser);
+
+        OrderService orderService = (OrderService) injector.getInstance(OrderService.class);
+        System.out.println(orderService.completeOrder(shoppingCartService.getByUser(exampleUser)));
+        shoppingCartService.addSession(yesterdayMovieSession, exampleUser);
+        System.out.println(shoppingCartService.getByUser(exampleUser));
+        System.out.println(orderService.completeOrder(shoppingCartService.getByUser(exampleUser)));
+        orderService.getOrdersHistory(exampleUser);
```

Comment:
The `orderService.getOrdersHistory(exampleUser);` method is called, but its result is not printed or logged. If this is intended for debugging or informational purposes, consider outputting the result to the console or a log to verify the user's order history.